components/RulesHeader/index.js

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+
+export default function RulesHeader() {
+  return (
+    <>
+      <StyledHeaderWrapper>
```

Comment:
I want to mention here:
This is a good example why clear userstories are valueable -> In your userstory the wireframe shows the + button, but it is not needed since for this US it has no effect. In my opinion the US should not show it and you should not think about it while working on this feature. As I see in your code you prepare for 'having this button in the future'.